docs/src/filters/writing_custom_filters.md

```diff
@@ -392,17 +387,16 @@ However, it usually contains a Protobuf equivalent of the filter's static config
        }
        ```
 
-[Filter]: #
-[FilterFactory]: #
-[filter-factory-name]: #FilterFactory::name
-[FilterRegistry]: #
-[FilterChain]: #
-[runner]: #
-[create-filter-args-config]: #CreateFilter::config
-[config-type-dynamic]: #ConfigType::Dynamic
+[Filter]: ../../../api/quilkin/filters/trait.Filter.html
+[FilterFactory]: ../../../api/quilkin/filters/trait.FilterFactory.html
+[filter-factory-name]: ../../../api/quilkin/0.1.0/quilkin/filters/trait.FilterFactory.html#tymethod.name
```

Comment:
```suggestion
[filter-factory-name]: ../../../api/quilkin/filters/trait.FilterFactory.html#tymethod.name
```

Don' t need to specify the version, it all gets bundled up under the the same parent directory, which has the version snapshot.